episodes/techniques-inbox-zero.md

```diff
@@ -0,0 +1,119 @@
+
+## What is Inbox Zero?
+
+Productivity guru [Merlin Mann](https://en.wikipedia.org/wiki/Merlin_Mann) began sharing the inbox zero method in the early 2000s as a way for people to “reclaim their inbox.” Based on David Allen’s [*Getting Things Done*](https://gettingthingsdone.com/) productivity system, inbox zero helps you keep your inbox under control so you can spend less time on email and more time focused on your work or other responsibilities.
+
+![](https://lh6.googleusercontent.com/U_MG_N27tRdDZ_gL_-PkfiBJl5QLw0bx6Sy4yjFJYbpOs_3IuNx9sXh67Mt4xe-lc15eCxUYLHZ7jMHP-9jk6-k5goxSKWnMYvlOTjWMV3RGRmirnwEVsIt_uMfAO6pRplUBGkR7qNAPI18wn3ZaQyY)
+
+In this carpentry episode it is important to recognise that the inbox zero method doesn't only have to be used to increase productivity. This technique can be used purely for improving mental health at work by:
+
+ 1. Reducing the burden of nagging emails.
+ 2. Freeing up your mind for other tasks.
+ 3. Becoming more organised.
+
+
+Instead of covering all aspects of `inbox zero` upfront, we will leave you to do further reading later, if you feel this technique could help you.  
+https://clean.email/blog/productivity/what-is-zero-inbox  
+https://blog.superhuman.com/inbox-zero-method/  
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+
+Inline instructor notes can help inform instructors of timing challenges
+associated with the lessons. They appear in the "Instructor View"
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Practice 1: Value Your Time
+
+The Pareto principle, also known as the 80/20 rule, states that roughly 80 percent of the effects come from 20 percent of the causes. This rule can also be applied to email management as only about 20 percent of emails deserve our attention. Luckily, many parts of the zero inbox method help you value your time and use it more effectively.
+
+Because email notifications both create a distraction and often go ignored, we recommend you **turn off all email notifications**. Instead, dedicate short, specific time slots throughout the day for email management. This helps you value your time and focus your attention on your inbox without distractions so you can organize your messages quickly.
+
+```
+Why not try right now to turn off email notifications. If you are using Windows 11 you can enable `do not disturb` mode like this:  
+![](https://winpoin.com/wp-content/uploads/2022/11/Cara-Cepat-Mengaktifkan-Do-Not-Disturb-Mode-di-Windows-11-2-768x392.jpg)
+
+```
+
+:::::::::::::::::::::::: solution 
+
+## Output
+ 
+```output
+[1] "This new lesson looks good"
+```
+
+:::::::::::::::::::::::::::::::::
+
+
+## Practice 2: Delete Junk e-mails
+
+:::::::::::::::::::::::: solution 
+
+You can add a line with at least three colons and a `solution` tag.
+
+:::::::::::::::::::::::::::::::::
+::::::::::::::::::::::::::::::::::::::::::::::::
```

Comment:
I guess the rest from this point is not needed, right?

Reply:
Yeah, you are right. This should have been removed

Reply:
@matscorse will do!